openhtf/util/measurements.py

```diff
@@ -203,16 +204,9 @@ def with_validator(self, validator):
 
   def with_args(self, **kwargs):
     """Creates a new Measurement, see openhtf.PhaseInfo.with_args."""
-    new_meas = mutablerecords.CopyRecord(self)
-    if '{' in new_meas.name:
-      formatter = lambda x: x.format(**kwargs) if x else x
-    else:
-      # str % {'a': 1} is harmless if str doesn't use any interpolation.
-      # .format is as well, but % is more likely to be used in other contexts.
-      formatter = lambda x: x % kwargs if x else x
-    new_meas.name = formatter(self.name)
-    new_meas.docstring = formatter(self.docstring)
-    return new_meas
+    return mutablerecords.CopyRecord(
+        self, name=util.format_string(self.name),
```

Comment:
i think you need to pass kwargs through to format_string?  in the other places you call format_string, too.